### PR TITLE
PFHub Upload: moose_itg_4a

### PR DESCRIPTION
**Pull request by staticmanapp**
This a new PFHub upload

To review:

- Wait for the tests to pass.

- Go to http://random-cat-{pull-request-number}.surge.sh/simulations/{simulation_name}

| Field     | Content                         |
| --------- | ------------------------------- |
| metadata  | [object Object]                 |
| benchmark | [object Object]                 |
| data      | [object Object],[object Object] |
| date      | 1534781873                      |

---
This change is [Reviewable](https://reviewable.io/reviews/usnistgov/pfhub/820)

### CODE REVIEW

**Comment by wd15** [comment]
@itgreenquist, thanks for this upload. Unfortunately, I haven't created a problem 4 landing page yet. I need to work on that. I can use your data to help with that though. Basically, this will take some time, maybe a few weeks.

Also you didn't include any data with this upload.

File: _data/simulations/moose_itg_4a/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+    cpu_architecture: x86_64
+    acc_architecture: none
+    parallel_model: distributed
+    clock_rate: '0'
+    cores: '8'
+    nodes: '1'
+benchmark:
+  id: 4a
+  version: '1'
+data:
```
Comment (wd15) [inline _data/simulations/moose_itg_4a/meta.yaml]: Need some data files

**Comment by itgreenquist** [comment]
@wd15,

I just saw that. I'll go ahead and try to upload the results again.

**Comment by wd15** [comment]
@itgreenquist, awesome, thanks. I'll let you know when I have something to show. I may also need to iterate a bit with you on this one. I'm not certain what data I need either.

**Comment by itgreenquist** [comment]
I submitted again, this time with data. Go ahead and delete this PR.

**Pull request closed without merging**